flight/PiOS/Common/pios_openlrs.c

```diff
@@ -104,6 +104,54 @@ const uint8_t default_hop_list[] = {DEFAULT_HOPLIST};
 const struct rfm22_modem_regs bind_params =
 { 9600, 0x05, 0x40, 0x0a, 0xa1, 0x20, 0x4e, 0xa5, 0x00, 0x20, 0x24, 0x4e, 0xa5, 0x2c, 0x23, 0x30 };
 
+static uint32_t minFreq(HwSharedRfBandOptions band) {
+	switch (band) {
+		default:
```

Comment:
and if you remove the board default value below you could just trash the default here so we get a compile failure if not covering any cases in the future.